data_science/palette/notebooks/00 - RGB, LAB, and human colour perception.ipynb

```diff
@@ -0,0 +1,163 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# RGB and CIE\n",
+    "In this project we'll regularly need to calculate distances between pixels in colour space as a proxy for the visual difference between the colours. The simplest way of doing this is to calculate the Euclidean distance between them (also known as cosine distance or $\\ell2$ norm).\n",
+    "If we have two colours $C_1 = (R_1, G_1, B_1)$ and $C_2 = (R_2, G_2, B_2)$, the Euclidean distance $\\Delta C$ is defined as:\n",
```

Comment:
Not sure what a 'cosine distance' is?

Reply:
cosine similarity, cosine distance, euclidean distance, euclidean similarity, l2 norm, RMSD, RMSE, square root of sum of squared differences, etc etc etc... all names for the same thing

Reply:
ok, I thought cosine similarity and Euclidean distance were different.

Reply:
Ack, you're totally right, not sure what was going on in my head when I wrote this!